fw/http_parser.c

```diff
@@ -7249,6 +7271,7 @@ do {									\
 	SET_HDR_CACHE_CONTROL_FIELD(max_age, TFW_HTTP_CC_MAX_AGE);
```

Comment:
We should add (req->cache_ctl.flags & TFW_HTTP_CC_STALE_IF_ERROR) for consistency